src/modules/UI/components/WalletListModal/components/WalletListModalBody.ui.js

```diff
@@ -95,10 +95,10 @@ export default class WalletListModalBody extends Component<$FlowFixMeProps> {
 
     // need to crossreference tokensEnabled with nativeBalances
     let enabledNativeBalances = {}
-    const enabledTokens = guiWallet.tokensEnabled
+    const enabledTokens = guiWallet.enabledTokens
 
     for (let prop in guiWallet.nativeBalances) {
-      if ((prop !== currencyCode) && enabledTokens[prop] && enabledTokens[prop].enabled) {
+      if ((prop !== currencyCode) && (enabledTokens.indexOf(prop) >= 0)) {
```

Comment:
if checking for the existence of a element in an array, please use [Array.prototype.includes](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes)
`[1, 2, 3].includes(1) === true`